packages/canon/src/components/Menu/types.ts

```diff
@@ -13,7 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import { Menu as MenuPrimitive } from '@base-ui-components/react/menu';
+import {
+  ForwardRefExoticComponent,
+  RefAttributes,
+  ComponentProps,
+} from 'react';
+
+/** @public */
+export type MenuComboboxOption = {
+  label: string;
+  value: string;
+  disabled?: boolean;
+};
+
+/** @public */
+export interface MenuComboboxProps extends ComponentProps<'div'> {
+  options: MenuComboboxOption[];
+  value?: string[];
+  onValueChange?: (value: string[]) => void;
+  multiselect?: boolean;
+  closeParentOnEsc?: boolean;
```

Comment:
I don't see this one used anywhere? Same thing with the existing definition of it actually.

Reply:
These are used in the `Combobox.tsx` file.

Reply:
Not that I can see O_o